Refactor README and update install instructions

nicdgonzalez opened the pull request:
Issue #4 was totally on me, I'm not sure how I missed needing to change the language setting, but I wanted to update the README to include the image you sent in [this comment](https://github.com/keve1227/furigana/issues/4#issuecomment-2347666052).

While I was at it, I also included additional installation instructions which provide options for moving the downloaded file via the command line.

I also threw in an extra showcase photo, just because :)

Let me know if you want me to make any changes!

keve1227 commented:
Thank you for contributing! I apologize for not being more on top of things. I will review your pull request and merge it sometime next week at the latest.

nicdgonzalez committed:
docs: refactor README and update install instructions

nicdgonzalez committed:
style: update width on showcase images

nicdgonzalez committed:
fix: remove quotes on first arg in Linux install command

Wrapping the first argument in double quotes prevents the wildcards from expanding to match the Furigana ZIP file's name.

nicdgonzalez committed:
docs: clarify Windows install instructions

Add notes for using Command Prompt and Powershell
